jwst/assign_wcs/nirspec.py

```diff
@@ -539,15 +566,25 @@ def get_open_msa_slits(msa_file, msa_metadata_id, slit_y_range=[-.55, .55]):
 
         # Not allowed....
         else:
-            raise ValueError("MSA configuration file has more than 1 shutter with "
-                             "sources for metadata_id = {}".format(msa_metadata_id))
+            message = ("MSA configuration file has more than 1 shutter with "
+                       "sources for metadata_id = {}".format(msa_metadata_id))
+            log.info(message)
+            raise MSAFileError(message)
```

Comment:
I wonder if we should just report this as an error/warning for the given slitlet in which it occurs and remove that slitlet from the list, but then proceed with the rest of processing (rather than terminating with an exception).

Reply:
I considered this but decided against since it will hide errors in APT or SDP as this should never happen. However, I'm happy to change it if you think that's what needs to happen.

Reply:
We can leave it as-is for now, and if we get complaints, it can be changed in the future.